proposals/000-comm-grants.md

```diff
@@ -0,0 +1,223 @@
+  efficiency investments where a small contribution can make a big difference
+  because someone is in a place where they are blocked from contributing.  This
+  doesn't necessarily have to be a top priority for the community.  If a small
+  investment can help even a second-tier priority, it is still a good thing.
+* We hope to build more community support and awareness of people contributing
+  to Haskell.  Bringing the community together in support of a shared goal can
+  clear the path of procedural obstacles.
+
+## Goals
+
+The goal of this proposal is to set up a small grant program administered by the
+Haskell Foundation.
+
+Submissions to the grant program should come from individuals who feel they
+could perform additional work on projects of benefit to the Haskell community if
+they had extra funding.  Submissions may include:
+
+* Open source tools and libraries.  This should focus on tools that are broadly
+  useful for Haskell developers.  Simply being written in Haskell is not enough.
+* Documentation and learning materialsthat are freely available to the
+  community.
+* Project leadership and support, including user support, project management,
+  bug triage and coordination, code review, and technical writing.
+
+Certain submissions are not appropriate, and those submitting them would be
+referred to the HFTP process, instead.  This includes anything asking for
+financial support above $2000.  (This is less than other similar programs, but
+we feel it's important to limit the risk given the lower standard of review
+involved.)  It also includes any submission that might be controversial in any
+way, because the more private nature of these proposals isn't the right process
+for anything potentially controversial.  Submissions should also not ask for
+assistance from the Haskell Foundation beyond the grant money, so if someone
+wants to recruit contributors, find infrastructure, etc. that they don't intend
+to pay for themselves, this is not the right process for that.  (However, a
+submission that explains that someone has already lined up extra assistance is
+welcome.)  Grants must be awarded to a single individual, and not shared among
+multiple contributors.  More complex proposals like this are also better suited
+the the HFTP process.
+
+These individuals would write a short summary of their intended work, limited to
+two pages, and answering the following questions:
+
+* What are you intending to do, and how will it benefit the Haskell community?
+* Why are you the right person to do this work?  If possible, submissions are
+  strongly encouraged to include a statement of support from an existing
+  maintainer of or core contributor to the project involved.
+* How much funding are you asking for, and how will it make your work possible?
+
+As a prerequisite for receiving a grant, individuals will agree to write two
+blog posts and share a link to discourse.haskell.org.  The first is a project
```

Comment:
Do we want these to be individual blog posts on some personal site, or on the Medium instance (https://haskell-foundation.medium.com/)? I'm inclined to say "let them choose the venue".